class PrimeRestoreException(Exception):
    pass


class PrimeRestoreUserException(PrimeRestoreException):
    pass
